src/components/app/Root.js

```diff
@@ -24,7 +24,30 @@ class Root extends React.Component {
                     path="/campaign-configurator"
                     render={props => <CampaignConfigurator d2={d2} {...props} />}
                 />
-
+                <Route
+                    path="/data-entry"
+                    component={() => {
+                        window.location =
+                            "http://dev2.eyeseetea.com:8082/dhis-web-dataentry/index.action";
+                        return null;
```

Comment:
To avoid hardcoding it, maybe can we use the react environment variable? something like `process.env.REACT_APP_DHIS2_URL`  (to be tested)

Reply:
By this you mean to assume that variable is going to be passed by command line when starting the app? or defining it inside the .env file? both ways are tested and work